features/machine/machine.feature

```diff
@@ -240,19 +239,42 @@ Feature: Machine features testing
-    Then admin ensures "default-valued-32269" machineset is deleted after scenario
+    Then admin ensures "<name>" machineset is deleted after scenario
+
+    Given I obtain test data file "cloud/ms-aws/<file_name>"
+    When I run oc create over "<file_name>" replacing paths:
+      | n                                                                                         | openshift-machine-api                       |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-cluster"]           | <%= infrastructure("cluster").infra_name %> |
+      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-machineset"]        | <name>                                      |
+      | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-cluster"]    | <%= infrastructure("cluster").infra_name %> |
+      | ["spec"]["template"]["spec"]["providerSpec"]["value"]["ami"]["id"]                        | <%= cb.default_ami_id %>                    |
+      | ["spec"]["template"]["spec"]["providerSpec"]["value"]["placement"]["availabilityZone"]    | <%= cb.default_availability_zone %>         |
+      | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-machineset"] | <name>                                      |
+      | ["metadata"]["name"]                                                                      | <name>                                      |
 
-    Given I obtain test data file "cloud/ms-aws/ms_default_values.yaml"
-    When I run oc create over "ms_default_values.yaml" replacing paths:
-      | n                                                                                         | openshift-machine-api                           |
-      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-cluster"]           | <%= infrastructure("cluster").infra_name %>     |
-      | ["spec"]["selector"]["matchLabels"]["machine.openshift.io/cluster-api-machineset"]        | default-valued-32269                            |
-      | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-cluster"]    | <%= infrastructure("cluster").infra_name %>     |
-      | ["spec"]["template"]["spec"]["providerSpec"]["value"]["ami"]["id"]                        | <%= cb.default_ami_id %>                        |
-      | ["spec"]["template"]["spec"]["providerSpec"]["value"]["placement"]["availabilityZone"]    | <%= cb.default_availability_zone %>             |
-      | ["spec"]["template"]["metadata"]["labels"]["machine.openshift.io/cluster-api-machineset"] | default-valued-32269                            |
     Then the step should succeed
 
+    And I wait up to 120 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource | machine |
+    Then the step should succeed
+    And the output should contain:
+      | Provisioned |
```

Comment:
Why would we expect a state `Provisioned` instead of `Running`?

Reply:
I have the same question, I just found this on gcp and wanted to raise a bug, machine stucks in Provisioned status if we created it using default value.

Reply:
We can change it to running  , I just wanted to save time for this , as other scenarios we already make sure machine is running ..(as we mostly have IPI deployments for test)  here main aim was to make sure machineset is able to populate default values and webhooks are working fine for machineset to be able to get created ..
Let me know your thoughts ..

Reply:
I think we need to make sure the machine could be running, on gcp machine will stucks in Provisioned status if we create it using default value, 
test result: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/218926/console
fix pr: https://github.com/openshift/verification-tests/pull/1733/files

Reply:
made change for validating for Running status